ui/src/app/views/workspace-about.tsx

```diff
@@ -2,151 +2,276 @@ import {Component} from '@angular/core';
-import {workspacesApi} from 'app/services/swagger-fetch-clients';
-import colors from 'app/styles/colors';
-import {reactStyles, ReactWrapperBase, withCurrentWorkspace, withUserProfile} from 'app/utils';
-import {sliceByHalfLength} from 'app/utils/index';
-import {navigate} from 'app/utils/navigation';
+import {cdrVersionStore, currentWorkspaceStore, serverConfigStore} from 'app/utils/navigation';
 import {WorkspaceData} from 'app/utils/workspace-data';
-import {WorkspacePermissions} from 'app/utils/workspace-permissions';
-import {Authority, Profile, SpecificPopulationEnum} from 'generated/fetch';
-import {ResearchPurposeDescription, ResearchPurposeItems, specificPopulations} from './workspace-edit';
+
+import {profileApi, workspacesApi} from 'app/services/swagger-fetch-clients';
+
+import {Button, Link} from 'app/components/buttons';
+import {InfoIcon} from 'app/components/icons';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TooltipTrigger} from 'app/components/popups';
+import {Spinner} from 'app/components/spinners';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {reactStyles, ReactWrapperBase, withUrlParams, withUserProfile} from 'app/utils';
+import {ResearchPurpose} from 'app/views/research-purpose';
+import {ResetClusterButton} from 'app/views/reset-cluster-button';
+import {CdrVersion, Cohort, Profile, UserRole, WorkspaceAccessLevel} from 'generated/fetch';
+import {WorkspaceShare} from './workspace-share';
+
+
+interface WorkspaceState {
+  sharing: boolean;
+  cdrVersion: CdrVersion;
+  useBillingProjectBuffer: boolean;
+  freeTierBillingProject: string;
+  cohortsLoading: boolean;
+  cohortsError: boolean;
+  cohortList: Cohort[];
+  workspace: WorkspaceData;
+  workspaceUserRoles: UserRole[];
+  googleBucketModalOpen: boolean;
+}
 
 const styles = reactStyles({
-  mainHeader: {
-    fontSize: '20px', fontWeight: 600, color: colors.primary, marginBottom: '0.5rem',
-    display: 'flex', flexDirection: 'row'
+  mainPage: {
+    display: 'flex', justifyContent: 'space-between', alignItems: 'stretch',
+    height: 'calc(100% - 60px)'
   },
-  sectionHeader: {
-    fontSize: '16px', fontWeight: 600, color: colors.primary, marginTop: '1rem'
+  rightSidebar: {
+    backgroundColor: colorWithWhiteness(colors.primary, 0.85), marginRight: '-0.6rem',
+    paddingLeft: '0.5rem', paddingTop: '1rem', width: '22%', display: 'flex',
+    flexDirection: 'column'
   },
-  sectionText: {
-    fontSize: '14px', lineHeight: '24px', color: colors.primary, marginTop: '0.3rem'
+  shareHeader: {
+    display: 'flex', flexDirection: 'row', justifyContent: 'space-between', paddingRight: '0.5rem',
+    paddingBottom: '0.5rem'
+  },
+  infoBox: {
+    backgroundColor: colorWithWhiteness(colors.primary, 0.75), height: '2rem', width: '6rem',
+    borderRadius: '5px', padding: '0.4rem', marginRight: '0.5rem', marginBottom: '0.5rem',
+    color: colors.black, lineHeight: '14px'
```

Comment:
I think the color should probably be `colors.primary`? If it is text?

Reply:
👍